### PR TITLE
Moving draft content to GitHub

### PR DESCRIPTION
**Pull request by JoanneOFlynn2018**
Moving draft content to GitHub

### CODE REVIEW

File: doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-rule-in-alertmanager.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
```
Comment (mrunge) [inline doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-rule-in-alertmanager.adoc]: is this file included twice?

File: doc-Service-Telemetry-Framework/modules/proc_preparing-your-openshift-environment-for-stf.adoc [hunk 2]
```diff
@@ -23,34 +23,4 @@
...
-----
-
-[WARNING]
-====
-If you want to deploy ElasticSearch, as the cluster administrator, you must set `vm.max_map_count` sysctl to `262144` on the host level of each node in your cluster before you instantiate an {ProjectShort} instance.
-----
-sudo sysctl -w vm.max_map_count=262144
-----
-====
+Catalogue source work and validate that ElasticSearch can launch based on something.
```
Comment (mrunge) [inline doc-Service-Telemetry-Framework/modules/proc_preparing-your-openshift-environment-for-stf.adoc]: needs an update

File: doc-Service-Telemetry-Framework/modules/proc_prerequisites-for-stf-deployment.adoc [hunk 3]
```diff
@@ -21,21 +21,17 @@
...
 
 Complete the following prerequisite tasks:
 
-. Install a suitable platform on which to install {Openshift} ({OpenShiftShort}), for
-example,
-link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/[Red
-Hat OpenStack Platform]. If you already have an {OpenShiftShort} 3.11
-environment, see
-<<preparing-your-openshift-environment-for-stf_installing-the-core-components-of-stf>>.
+. Deploy OCP. For more information, see https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/
```
Comment (mrunge) [inline doc-Service-Telemetry-Framework/modules/proc_prerequisites-for-stf-deployment.adoc]: . Deploy {Openshift}

File: doc-Service-Telemetry-Framework/modules/proc_prerequisites-for-stf-deployment.adoc [hunk 4]
```diff
@@ -21,21 +21,17 @@
...
+. Deploy {ProjectShort} to the {OpenShiftShort} environment. For more information, see <<deploying-stf-to-the-openshift-environment_installing-the-core-components-of-stf>>.
+. Configure OpenStack templates and deploy changes into cloud infrastructure. For more information, see Completing the STF configuration.
```
Comment (mrunge) [inline doc-Service-Telemetry-Framework/modules/proc_prerequisites-for-stf-deployment.adoc]: replace STF with {ProjectShort}

File: doc-Service-Telemetry-Framework/modules/con_alerts.adoc [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+= Alerts
+
+You can configure alerts to receive notifications, for example, when the load average of a node gets higher than 1.0 or when a disk is almost full.
+
+For alerting, two components are involved: First of all, Prometheus executes rules. They can be found in Prometheus web UI under Status/Rules. These rules are evaluated, and if the expression is true, an alert triggers. Alerts are also shown under the Alerts tab in the web UI.
+
+To create an alert, complete the following tasks:
+
+. Create an alert rule in Prometheus. For more information, see Creating a rule in Prometheus.
+. Create an alert route in Alertmanager. For more information, see Creating an alert route in Alertmanager.
```
[the remark this answers is not in the document]
Comment (JoanneOFlynn2018) [reply doc-Service-Telemetry-Framework/modules/con_alerts.adoc]: Ok I'll update that, I had them down as 2 separate things but i'll put that content together

File: doc-Service-Telemetry-Framework/modules/proc_preparing-your-openshift-environment-for-stf.adoc (hunk 2 above)
Comment (JoanneOFlynn2018) [reply doc-Service-Telemetry-Framework/modules/proc_preparing-your-openshift-environment-for-stf.adoc]: Yes this is just a placeholder sentence to remind Leif.

File: doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-rule-in-alertmanager.adoc (hunk 1 above)
Comment (JoanneOFlynn2018) [reply doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-rule-in-alertmanager.adoc]: Committed

File: doc-Service-Telemetry-Framework/modules/proc_prerequisites-for-stf-deployment.adoc (hunk 3 above)
Comment (JoanneOFlynn2018) [reply doc-Service-Telemetry-Framework/modules/proc_prerequisites-for-stf-deployment.adoc]: Committed

File: doc-Service-Telemetry-Framework/modules/proc_prerequisites-for-stf-deployment.adoc (hunk 4 above)
Comment (JoanneOFlynn2018) [reply doc-Service-Telemetry-Framework/modules/proc_prerequisites-for-stf-deployment.adoc]: Committed

File: doc-Service-Telemetry-Framework/modules/con_the-core-components-of-stf.adoc [hunk 6]
```diff
@@ -23,43 +23,11 @@
...
-ifeval::["{build}" == "downstream"]
-. Create an RHCC Secret. For more information, see <<creating-an-rhcc-secret_installing-the-core-components-of-stf>>.
-endif::[]
-
-. Generate a TLS certificate for {MessageBus}. For more information, see
-<<generating-a-tls-certificate-for-amq-interconnect_installing-the-core-components-of-stf>>.
-
-. Generate TLS certificates for ElasticSearch. For more information, see
-<<generating-tls-certificates-for-elasticsearch_installing-the-core-components-of-stf>>.
+* Apache Qpid Dispatch Router
```
[the remark this answers is not in the document]
Comment (JoanneOFlynn2018) [reply doc-Service-Telemetry-Framework/modules/con_the-core-components-of-stf.adoc]: Committed

File: doc-Service-Telemetry-Framework/modules/con_architecture.adoc [hunk 7]
```diff
@@ -50,3 +50,9 @@ layers:
 * Infrastructure platform
 
 image::STF_Overview_37_0819_deployment_prereq.png[]
+
+For more information about how to deploy OpenShift Container Platform, see https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/[Product Documentation] for OpenShift Container Platform 4.3. You can install {OpenShift} on cloud platforms or on bare metal.
+For more information about {ProjectShort} performance and scaling, see https://access.redhat.com/articles/4907241.
+
+NOTE:
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/con_architecture.adoc]: Invalid formatting. Should be `[NOTE]`

File: doc-Service-Telemetry-Framework/modules/con_introduction-to-stf.adoc [hunk 8]
```diff
@@ -22,17 +22,8 @@
...
-
-
-{Project} ({ProjectShort}) is an application that runs on the {OpenShift}
-({OpenShiftShort}). Use {ProjectShort} to collect metrics and record events
-from the nodes in your systems that you want to monitor. The metrics and event
-information travels on a message bus to the server side for storage. Use this
-centralized information as the source for alerts, visualization, or the source
-of truth for orchestration frameworks.
+{Project} ({ProjectShort}) is an application that runs on {OpenShift} ({OpenShiftShort}). Use {ProjectShort} to collect metrics and record events from the nodes in your systems that you want to monitor. The metrics and event information travels on a message bus to the server side for storage. Use this centralized information as the source for alerts, visualization, or the source of truth for orchestration frameworks.
+{ProjectShort} provides metrics and events data collection from OpenStack infrastructure, fast and reliable transport of data, and built-in data storage and alerting capabilities. An advantage over other monitoring systems is its lighter weight and modular design.
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/con_introduction-to-stf.adoc]: s/OpenStack/{OpenStack}/  ?

File: doc-Service-Telemetry-Framework/modules/proc_adding-an-alert-route-to-alertmanager.adoc [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+    receivers:
+    - name: 'webhook'
+      webhook_configs:
+      - url: http://192.168.36.9/webhook
+    - name: 'webhook-crit'
+      webhook_configs:
+      - url: http://192.168.36.9/webhook-crit
+----
++
+The example in the previous step creates two receivers to be called on different severities. Edit the parameters in the example to suit your requirements. For more information, see https://prometheus.io/docs/alerting/configuration/.
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/proc_adding-an-alert-route-to-alertmanager.adoc]: Still learning the style guide, but "The example in the previous step" is generally frowned upon right? (the part that refers to a previous section?)

File: doc-Service-Telemetry-Framework/modules/proc_adding-an-alert-route-to-alertmanager.adoc [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
++
+----
+oc project service-telemetry
+----
+. Run the following command:
++
+----
+oc delete secret alertmanager-stf-default
+----
+. Create the new Alertmanager configuration secret. Create a file named alerting-manager-config.yaml. Copy the contents of the following example into the new file.
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/proc_adding-an-alert-route-to-alertmanager.adoc]: alerting-manager-config.yaml

Needs to be wrapped in backticks

File: doc-Service-Telemetry-Framework/modules/proc_adding-an-alert-route-to-alertmanager.adoc [hunk 11]
```diff
@@ -0,0 +1,108 @@
...
+. Log in to OpenShift.
+. Navigate to the service-telemetry namespace:
++
+----
+oc project service-telemetry
+----
+. Run the following command:
++
+----
+oc delete secret alertmanager-stf-default
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/proc_adding-an-alert-route-to-alertmanager.adoc]: @mrunge this isn't managed by the operator?

File: doc-Service-Telemetry-Framework/modules/proc_adding-an-alert-route-to-alertmanager.adoc [hunk 12]
```diff
@@ -0,0 +1,108 @@
...
+oc apply -f <filename>
+----
+. Restart Alertmanager:
++
+----
+oc delete pod alertmanager-stf-default-0
+----
+
++
+The Prometheus Operator restarts Alertmanager automatically.
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/proc_adding-an-alert-route-to-alertmanager.adoc]: If the operator restarts alertmanager automatically, then why do we delete the pod? I'm confused by the approach here.

File: doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-route-in-alertmanager.adoc [hunk 13]
```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-route-in-alertmanager.adoc]: I read the same text earlier in this commit. Do we have duplication of content here?

File: doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-rule-in-prometheus.adoc [hunk 14]
```diff
@@ -0,0 +1,57 @@
...
+// reused multiple times in a guide.
+//
+// Start the title with a verb, such as Creating or Create. See also
+// _Wording of headings_ in _The IBM Style Guide_.
+[id="creating-an-alert-rule-in-prometheus_{context}"]
+= Creating an alert rule in Prometheus
+
+Prometheus evaluates alert rules to trigger notifications. If the rule condition returns an empty result set, the condition is false. Otherwise, the rule is true and it triggers an alert. To create an alert rule, complete the following steps:
+
+. Create a yaml file for the alert rule. The following example in qpid-router-down.yaml is an example of a small, efficient alerting rule:
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-rule-in-prometheus.adoc]: qpid-router-down.yaml

backticks

File: doc-Service-Telemetry-Framework/modules/proc_estimating-storage-requirements-for-prometheus.adoc [hunk 15]
```diff
@@ -0,0 +1,35 @@
...
+//
+// Start the title with a verb, such as Creating or Create. See also
+// _Wording of headings_ in _The IBM Style Guide_.
+[id="estimating-storage-requirements-for-prometheus_{context}"]
+= Estimating storage requirements for Prometheus
+
+On average, Prometheus uses only around 1-2 bytes per sample. To plan the capacity of a Prometheus server, you can use this formula:
+`needed_disk_space = retention_time_seconds * ingested_samples_per_second * bytes_per_sample`
+
+NOTE:
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/proc_estimating-storage-requirements-for-prometheus.adoc]: s/NOTE:/[NOTE]/g

File: doc-Service-Telemetry-Framework/modules/proc_estimating-storage-requirements-for-prometheus.adoc [hunk 16]
```diff
@@ -0,0 +1,35 @@
...
+The result of this formula is an approximate size for the Prometheus server.
+
+In default settings, the retention time is 86400 seconds, and we are assuming 2 bytes per sample.
+
+NOTE:
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/proc_estimating-storage-requirements-for-prometheus.adoc]: NOTE: --> [NOTE]

File: doc-Service-Telemetry-Framework/modules/proc_operating-the-dashboard.adoc [hunk 17]
```diff
@@ -0,0 +1,178 @@
...
+// Start the title with a verb, such as Creating or Create. See also
+// _Wording of headings_ in _The IBM Style Guide_.
+[id="operating-the-dashboard_{context}"]
+= Operating the dashboard
+
+The infrastructure dashboard shows metrics for a single node at a time. Nodes can be selected in the upper left hand corner of the dashboard.
+
+Sections can be expanded and collapsed by selecting the horizontal dividers.
+
+Panels
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/proc_operating-the-dashboard.adoc]: Feels like this is a title or something?

File: doc-Service-Telemetry-Framework/modules/proc_prerequisites-for-stf-deployment.adoc [hunk 18]
```diff
@@ -21,21 +21,17 @@
...
 
-. Install a suitable platform on which to install {Openshift} ({OpenShiftShort}), for
-example,
-link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/[Red
-Hat OpenStack Platform]. If you already have an {OpenShiftShort} 3.11
-environment, see
-<<preparing-your-openshift-environment-for-stf_installing-the-core-components-of-stf>>.
+. Deploy {OpenShift}. For more information, see https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/
+. Deploy {ProjectShort} to the {OpenShiftShort} environment. For more information, see <<deploying-stf-to-the-openshift-environment_installing-the-core-components-of-stf>>.
+. Configure {OpenStack} templates and deploy changes into the cloud infrastructure. For more information, see Completing the STF configuration.
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/proc_prerequisites-for-stf-deployment.adoc]: Need to change _STF_ to _{ProjectShort}_ I think

File: doc-Service-Telemetry-Framework/modules/proc_setting-up-openstack-on-the-client-side.adoc [hunk 19]
```diff
@@ -20,10 +20,9 @@
...
 // _Wording of headings_ in _The IBM Style Guide_.
-[id="setting-up-openstack-on-the-client-side_{context}"]
+[id="configuring-openstack-on-the-client-side_{context}"]
 = Configuring {OpenStack} on the client side
 
 To collect metrics, events, or both, and to send them back to the {ProjectShort}
 storage domain, you must install collectd and {MessageBus} on the nodes of an
-{OpenStack} deployment. Configure the {OpenStackLong} {OpenStackInstaller} to enable the data collection
-functionality and to stream the data to {Project} ({ProjectShort}).
+{OpenStack} deployment. If you want to deploy with routed L3 networks, see Deploying with routed L3 networks.
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/proc_setting-up-openstack-on-the-client-side.adoc]: the _see Deploying with routed L3 networks_ needs to be turned into a link

**Pull request merged**